virt/virt-2-6-release-notes.adoc

```diff
@@ -98,6 +98,9 @@ The SVVP Certification applies to:
 [id="virt-2-6-known-issues"]
 == Known issues
 
+* A virtual machine instance (VMI) fails to migrate if the `cdrom` drive is set to `readonly: true` in the VMI spec. The following message displays: `Operation not supported: Cannot migrate empty or read-only disk sdb`.
```

Comment:
The ISG says `Use display as a transitive verb only.` 

Suggestion: s/The following message displays/The following message is displayed